spec/required/users_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  context '#index' do
+    it "assigns all users to @users" do
+      get("/")
+      expect(assigns(:users)).to eq(User.all)
+    end
+    it 'render a template' do
+      get("/")
+      expect(response).to render_template('index')
+    end
+    it 'Response Status To Be 200' do
+      get("/")
+      expect(response).to have_http_status(200)
+    end
+  end
+
```

Comment:
- [ ]  Kindly add a test that ensures the response body includes the correct placeholder text. Also kindly ensure you do all tests for each individual view(index and show), in this case, you should have 6 tests or more.